.vsts-ci/windows.yml

```diff
@@ -69,6 +69,11 @@ stages:
       purpose: ElevatedPesterTests
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-macos`
Group policy settings tests.Group policy settings tests.Transcription policy test
```powershell
Cannot bind argument to parameter 'Path' because it is null.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/engine/Basic/GroupPolicySettings.Tests.ps1: line 170
170:             if (-not (Test-Path $KeyPath)) {$null = New-Item $KeyPath}

```